lighthouse-cli/package.json

```diff
@@ -8,6 +8,8 @@
     "typescript": "^2.0.3"
   },
   "dependencies": {
-    "@types/node": "^6.0.45"
+    "@types/node": "^6.0.45",
+    "express": "^4.14.0",
```

Comment:
until we need more advanced server features, could we limit this to a more simple server a la [static-server](https://github.com/GoogleChrome/lighthouse/blob/81194e4aa8594679483b9eb56d9b18b622f2d27e/lighthouse-cli/test/fixtures/static-server.js) to avoid the extensive dependencies `express` brings in until we need them?